src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres?useSSL=false
+spring.datasource.url=jdbc:postgresql://localhost:5440/postgres?useSSL=false
```

Comment:
Why you change port? If you use a docker, it is better to keep it on your computer, so that there will be no problems when they merge the branch into the master